wagtail/contrib/styleguide/views.py

```diff
@@ -95,58 +96,65 @@ def __init__(self, *args, **kwargs):
 icon_comment_pattern = re.compile(r"<!--!(.*?)-->")
 
 
-def index(request):
+class IndexView(WagtailAdminTemplateMixin, TemplateView):
+    template_name = "wagtailstyleguide/base.html"
```

Comment:
```suggestion
    template_name = "wagtailstyleguide/base.html"
    page_title = _("Styleguide")
```

The page title is missing, see screenshot below. Left is your current code, right is the `main` branch code.

<img width="450" alt="Screenshot 2023-10-27 at 7 37 30 am" src="https://github.com/wagtail/wagtail/assets/1396140/ba3cf5d3-89c2-4f3c-8602-ec1ddd730c6f">